src/Common/Auth/AuthUtils.php

```diff
@@ -268,7 +272,7 @@ private function confirmUserPassword($username, &$password)
         $ip = collectIpAddresses();
 
         // Check to ensure username and password are not empty
-        if (empty($username) || empty($password)) {
+        if ((empty($username) || empty($password))) {
```

Comment:
is this change meaningful?

Reply:
There might have been at some point mid-commit a different logic check.